install/installer/example-config.yaml

```diff
@@ -11,7 +11,6 @@ containerRegistry:
   privateBaseImageAllowList: []
 database:
   inCluster: true
-disableDefinitelyGp: true
```

Comment:
Removal of properties is alright? Or installer can fail in case the property still specified?

Reply:
Test with leeway dev:preview, that it can fail if we put some unknown field
<img width="954" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/370cd6ce-4269-446b-9377-74eb9574b5cb">

Reply:
We need to keep this `disableDefinitelyGp` configuration before we clean ops and Dedicated configs up